src/cloudsc_loki/cloudsc_cuf_loki.config

```diff
@@ -0,0 +1,44 @@
+derived_types = ['TECLDP']
+
+[dic2p]
+NLEV = 137
+
+[default]
+# Specifies the behaviour of auto-expanded routines
+role = 'kernel'
+expand = true  # Automatically expand subroutine calls
+strict = true  # Throw exceptions during dicovery
+
+# Ensure that we are never adding these to the tree, and thus
+# do not attempt to look up the source files for these.
+# TODO: Add type-bound procedure support and adjust scheduler to it
+disable = ['timer%start', 'timer%end', 'timer%thread_start', 'timer%thread_end',
```

Comment:
the first two lines with `timer%...` should be safe to remove since the scheduler knows now how to resolve the type name.